plugins/react.js

```diff
@@ -5,12 +5,23 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
+// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+const {
+  default: jsxDevTransform
+} = require('@babel/plugin-transform-react-jsx-development');
```

Comment:
I tried

```
const jsxDevTransform = require('@babel/plugin-transform-react-jsx-development').default;
```
and received

```
8:7   error    Unsafe assignment of an `any` value                  @typescript-eslint/no-unsafe-assignment
```
and 

```
8:25  error    Unsafe member access .default on an `any` value      @typescript-eslint/no-unsafe-member-access
```

Reply:
Removed in https://github.com/zendeskgarden/eslint-config/pull/181/commits/e568f39d835a95156a53749cd594478d244e1388